groups/p2p/peer_with_partners.js

```diff
@@ -18,20 +18,20 @@ const times = 2 * 60 * 30;
 
   @returns via cbk or Promise
 */
-module.exports = ({capacity, inbound, lnd, outbound}, cbk) => {
+module.exports = (args, cbk) => {
```

Comment:
why is this changed now? is it still referenced?

Reply:
Hmm, no. I referenced it before. Maybe I can just revert the file to the master branch.